internal/core/client/resolver.go

```diff
@@ -34,7 +35,7 @@ func (client *Client) ResolvePath(ctx context.Context, raw string) (*types.Resol
 
 	res.RepoName = parts[1]
 	res.Repository, err = client.GetRepository(ctx, orgID, res.RepoName)
-	if err != nil {
+	if err != nil && !errors.Is(err, types.ErrRepositoryNotExist) {
```

Comment:
What if it throws an error other than `ErrRepositoryNotExist`?

Reply:
in that case, it would return the error. however, now we're returning both &res, err so this is outdated